packages/esm-patient-immunizations-app/src/immunizations/components/immunizations-sequence-table.component.tsx

```diff
@@ -82,6 +83,14 @@ const SequenceTable: React.FC<SequenceTableProps> = ({
         noToday: true,
         time: false,
       }),
+    nextDoseDate:
+      (dose?.nextDoseDate &&
+        formatDate(new Date(dose.nextDoseDate), {
```

Comment:
It strikes me as strange that when loading this from FHIR, we're switching this to a String and then when calling `formatDate()` we're changing it to a date.